tcms/static/js/bugs.js

```diff
@@ -14,12 +14,10 @@ function loadBugs(selector, filter) {
             },
             {
                 data: null,
-                render: function (data, type, full, meta) {
-                    return `<a href="#bugs" data-toggle="popover" data-html="true"
+                render: (data, type, full, meta) => `<a href="#bugs" data-toggle="popover" data-html="true"
                         data-content="undefined" data-trigger="focus" data-placement="top">
                         <span class="fa fa-info-circle"></span>
-                        </a>`;
-                }
+                        </a>`
```

Comment:
This is unrelated, why change it ?

Reply:
because it's inconsistent with the other parts of the file